tasks/RedHat/generic/assert-epb.yml

```diff
@@ -56,13 +57,13 @@
         - name: Assert that EPB is configured at boot time (x86_64 platform only)
           assert:
             that: "__boot_local_cpupower_result.stdout == 'cpupower set -b 0'"
-            fail_msg: "FAIL: EPB is not configured at boot time!"
-            success_msg: "PASS: EPB is configured at boot time."
+            fail_msg: "FAIL: EPB is not configured to bet set to '0' ('performance') at boot time!"
+            success_msg: "PASS: EPB is configured to bet set to '0' ('performance') at boot time."
           ignore_errors: "{{ sap_hana_preconfigure_assert_ignore_errors|d(false) }}"
           when: __stat_boot_local.stat.exists
 
         - name: Get the current setting of EPB (x86_64 platform only)
-          command: cpupower info | grep perf-bias
+          command: bash -lc "cpupower info -b | awk '/perf-bias/{print $NF}'"
```

Comment:
use `shell` instead of `command: bash -lc` - the `shell` module is used instead of `command` when you need to use shell features such as pipes, redirection, variables, etc.